feat(OH-190): 이미지 업로드 로직 개발

rhrudska987 opened the pull request:
## 관련 이슈
https://onehunnit.atlassian.net/browse/OH-190

## 작업 내용
- url을 받아서 파일(JPEG)로 변환 후, NCP에 업로드 후, 반환

## 참고사항

rhrudska987 committed:
feat(OH-190): 이미지 업로드 로직 개발

## 관련 이슈
https://onehunnit.atlassian.net/browse/OH-190

## 작업 내용
- url을 받아서 파일(JPEG)로 변환 후, NCP에 업로드 후, 반환

## 참고사항

sjsjsj1246 commented on src/main/java/kr/co/onehunnit/onhunnit/domain/account/Account.java:
```diff
@@ -56,18 +56,19 @@ public class Account extends BaseTimeEntity {
 		this.phone = requestDto.getPhone();
 		this.gender = Gender.valueOf(requestDto.getGender());
 		this.birthday = requestDto.getBirthday();
+		this.profile_image = profile_image;
```
dto에 있는 거 아닌가요? 따로 파라미터로 받아야 하나요?

sjsjsj1246 commented on src/main/java/kr/co/onehunnit/onhunnit/service/AccountService.java:
```diff
@@ -28,16 +28,18 @@ public class AccountService {
 		Provider provider = Provider.valueOf(requestDto.getProvider());
 		Account account = accountRepository.findByEmailAndProvider(email, provider)
 			.orElseThrow(() -> new ApiException(ErrorCode.NOT_EXIST_EMAIL));
+		String profileImageUrl = imageUploadService.convertUrlToImage(requestDto.getProfile_image());
```
아 url을 받는다는게 뭐냐면. 외부 다른 곳의 이미지가 있고 그 이미지 url을 준다는게 아니라요. 저희 서버에서 업로드한 이미지의 url을 보낼거라는 뜻입니다. 그래서 따로 이걸 또 업로드할 필요가 없을 것 같아요. 그냥 이름 전화번호같이 등록만 하면 될거같습니다.

rhrudska987 committed:
feat(OH-190): 회원가입 시 이미지 url 저장

## 관련 이슈
https://onehunnit.atlassian.net/browse/OH-190

## 작업 내용
- url을 저장 후, 반환

## 참고사항
